library/Odesk/Phystrix/AbstractCommand.php

```diff
@@ -280,6 +287,25 @@ public function setServiceLocator(LocatorInterface $serviceLocator)
         $this->serviceLocator = $serviceLocator;
     }
 
+
+    /**
+     * Logic to record events and exceptions as they take place
+     *
+     * @param string    $eventName  type from class constants EVENT_*
+     * @param Exception $e          exception that was thrown as part of $eventName
+     */
+    private function triggerEvent($eventName, \Exception $e = null )
```

Comment:
the conversation we started regarding this seem to be gone, probably due to the updated PR. Sorry, I never got to responding to your comment.

1)
> he event and exception are actually tied together (one drove the other).

I don't really see why. Event and execution exception are not directly related. There can only be one execution exception per command. The exception is available everywhere as ->getExecutionException();.

2) I don't really see much of a use case for this, but I don't mind either. If you could remove exception from this method and rename the private one to recordExecutionEvent and the protected one to processExecutionEvent - I will merge it